pygeo/constraints/areaConstraint.py

```diff
@@ -508,7 +490,7 @@ def evalFunctions(self, funcs, config):
         self.p1 = self.DVGeo.update(self.name + "p1", config=config)
         self.p2 = self.DVGeo.update(self.name + "p2", config=config)
 
-        self.X = self._computeArea(self.p0, self.p1, self.p2, self.axis)
+        self.X = self._computeProjectedAreaTri(self.p0, self.p1, self.p2, self.axis)
```

Comment:
Is there a change in behavior here? Why is it the projected area now? Was it wrong before?

Reply:
(I think the issue is what I mentioned above, but let's wait for Neil's feedback)

Reply:
I renamed the function here because I felt `_computeArea` is too vague and generic for what it actually does, and also to distinguish with the other functions that previously were called `computeArea`.